app/tasks/raster_tile_cache_assets/symbology.py

```diff
@@ -313,14 +293,16 @@ async def date_conf_intensity_multi_8_symbology(
 
     # What we want is a value of 55 (max intensity for this scenario)
     # anywhere there is an alert in any system.
-    intensity_max_calc_string = f"np.ma.array(np.logical_or.reduce((A.data, B.data, C.data)) * {MAX_8_BIT_INTENSITY})"
+    intensity_max_calc_string = (
+        f"np.ma.array((A.data > 0) * {MAX_8_BIT_INTENSITY}, mask=False)"
+    )
 
     intensity_co = source_asset_co.copy(
         deep=True,
         update={
             "calc": None,
             "band_count": 1,
-            "data_type": DataType.uint8,
+            "data_type": DataType.uint16,
```

Comment:
Not familiar with this dataset yet, curious why the data type needs to change.

Reply:
Eh, good question. To be honest this is a change left over from when I was attempting to diagnose a problem. I knew that in some cases mixing data types in a VRT is problematic, though in this case it turns out not to matter (there's no problem promoting the uint8 to uint16 when read through the VRT with the other uint16 band). But uint8 takes up less space, so I'll change it back. Thanks for asking!